subcell_pipeline/analysis/compression_metrics/compression_analysis.py

```diff
@@ -208,21 +255,27 @@ def get_contour_length_from_trace(
 
 def get_bending_energy_from_trace(
     polymer_trace: np.ndarray,
-    **options: dict,
-) -> float:
+    **options: dict[str, Any],
+) -> Union[float, np.floating[Any]]:
     """
-    Returns the bending energy per monomer of a polymer trace.
+    Calculate the bending energy per monomer of a polymer trace.
 
     Parameters
     ----------
-    polymer_trace: [n x 3] numpy array
+    polymer_trace
         array containing the x,y,z positions of the polymer trace
-    **options: dict
+
+    **options
         Additional options as key-value pairs.
         bending_constant: float
             bending constant of the fiber
+
+    Returns
+    -------
+    :
+        bending energy per monomer of the polymer trace
     """
-    bending_constant = options.get("bending_constant", 1)
+    bending_constant = options.get("bending_constant", 1.0)
```

Comment:
For all instances of default values, define a constant (e.g. `DEFAULT_BENDING_CONSTANT = 1.0`) instead of using magic numbers.

Reply:
Fixed in [f8ffc9](https://github.com/simularium/subcell-pipeline/pull/58/commits/f8ffc9bc7f8a2e06701ffbdfdb6849aa836ee4b2)